__tests__/app.test.js

```diff
@@ -81,5 +81,43 @@ describe("4.GET /api/articles", () => {
         expect(articles).toBeSortedBy("created_at", { descending: true });
       });
   });
-  
+});
+describe("5.GET /api/articles/:article_id", () => {
+  test("responds with statusCode 200 and the response contains an article object ", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toHaveLength(1);
+        expect(article[0].article_id).toBe(2);
+        expect(article[0]).toEqual(
```

Comment:
once you change the model to return `results.rows[0]` that will remove the need for the `[0]` in these tests - and also remove the need for the check of `.length`